### PR TITLE
Map select transducers

### PR DESCRIPTION
**Pull request by AspieCoder1**
Added documentation for: 
- map transducer
- selection transducer

### CODE REVIEW

File: docs/pipeline/map.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Map transducer
+
+## How it works
+This transducer takes in all the data sources, relationships and similarity scores generated by profiling.
+Datapreparer then joins the data together to create combined data sources.
+To access this, click on the **mapping generation node** in the *latest wrangle result section*.
+
```
Comment (KenjiPcx) [inline docs/pipeline/map.md:8]: Maybe we can add an image here

File: docs/pipeline/map.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Here is the result of clicking *m02*:
+![m02-page](https://i.imgur.com/mjrGGDN.png)
+On this page, we can see:
+
+- **provenance** - the data sources used in the mapping
+- a diagram that shows the mapping as an input/output diagram.
+    Clicking on each element inside the diagram gives you the ability to view user context, data sources, attributes and properties.
+- a table view of the joined data
+  
+> If you want to explore the original data, you can click on a data source. > This will take you to a page to view the data source.
```
Comment (KenjiPcx) [inline docs/pipeline/map.md]: We can link this back to a viewing guide

**Review by KenjiPcx — COMMENTED** [review]
I think we should meet up and decide which style we want to follow, because I think me, you and Eiman have written it differently. Personally, I am also not sure if I included too much details in my docs.

File: docs/pipeline/map.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+## Viewing a mapping
+If you look at the output in the above example, you will see the mapping names highlighted.
+Clicking on this gives us some key details of the mapping.
+Here is the result of clicking *m02*:
+![m02-page](https://i.imgur.com/mjrGGDN.png)
+On this page, we can see:
+
+- **provenance** - the data sources used in the mapping
+- a diagram that shows the mapping as an input/output diagram.
+    Clicking on each element inside the diagram gives you the ability to view user context, data sources, attributes and properties.
```
Comment (AspieCoder1) [inline docs/pipeline/map.md:33]: Added a placeholder for the link

File: docs/pipeline/map.md (hunk 2 above)
Comment (AspieCoder1) [reply docs/pipeline/map.md]: Added empty link that can be filled in later

File: docs/pipeline/map.md (hunk 1 above)
Comment (AspieCoder1) [reply docs/pipeline/map.md:8]: Added image

File: docs/pipeline/map.md (hunk 3 above)
Comment (AspieCoder1) [reply docs/pipeline/map.md:33]: Added link for page

**Review by KenjiPcx — APPROVED** [review]
Everything is resolved

**Pull request merged**